uber/custom_tags.py

```diff
@@ -518,6 +518,21 @@ def render(self, context):
         return '<div class="prereg-price-notice">{}</div>'.format(notice) if notice else ''
 
 
+@tag
+class table_prices(template.Node):
+    def render(self, context):
+        if len(c.TABLE_PRICES) <= 1:
+            return '${} per table'.format(c.TABLE_PRICES['default_price'])
+        else:
+            cost, costs = 0, []
+            for i in range(1, 1 + c.MAX_TABLES):
```

Comment:
Yeah, it's inclusive on the first parameter and exclusive on the second parameter - just for extra confusion!

The reason why it works that way is because you'd commonly want to iterate over lists, so if you did something like
``for i in range(len(mylist)):``
then you'll get all of the list indexes.  Whereas if it was inclusive, you'd run off the end of the list.  And more generally "programmers start counting at zero" I guess :/